### PR TITLE
django conversion /plugins/importers/<importer_id>/

### PR DESCRIPTION
**Pull request by asmacdo**

### CODE REVIEW

**Comment by asmacdo** [comment]
`GET /v2/plugins/importers/yum_importer/` http://fpaste.org/173627/28872142/
`GET /v2/plugins/importers/notreal/` http://fpaste.org/173631/42202905/

File: server/pulp/server/webservices/views/plugins.py [hunk 1]
```diff
@@ -16,7 +17,34 @@ class DistributorsView(View):
...
+    def get(self, request, importer_id):
+        """
+        Return a serialized object representing the specified importer.
+
+        :param request : WSGI request object
+        :type  request : django.core.handlers.wsgi.WSGIRequest
+        :param importer_id : name of importer to return information for
+        :type  importer_id : string
+
+        :return : Serialized object representing the specified importer
```
Comment (mhrivnak) [inline server/pulp/server/webservices/views/plugins.py]: same as before, this object is not itself serialized.

File: server/test/unit/server/webservices/views/test_plugins.py [hunk 2]
```diff
@@ -17,7 +18,53 @@ class TestDistributorsView(unittest.TestCase):
...
+        Importer Resource should return a serialized object of the specified type
+        """
+        mock_manager = mock.MagicMock()
+        mock_manager.importers.return_value = [{'id': 'mock_importer_1'}, {'id': 'mock_importer_2'}]
+        mock_factory.plugin_manager.return_value = mock_manager
+        request = mock.MagicMock()
+        request.get_full_path.return_value = '/mock/path/'
+
+        importer_resource = ImporterResourceView()
+        importer_resource.get(request, 'mock_importer_2')
```
Comment (mhrivnak) [inline server/test/unit/server/webservices/views/test_plugins.py]: and as before, it would be helpful to assert that this actually returns something sane.

**Comment by mhrivnak** [comment]
merge when ready

**Pull request merged**